static/src/js/components/Preferences/PreferencesForm.jsx

```diff
@@ -41,10 +42,13 @@ const PreferencesForm = (props) => {
     radio: PreferencesRadioField
   }
 
-  const validate = (formDataObject, errors) => {
-    // Projections that aren't geographic have a zoom limit of 4
-    if (formDataObject.mapView.projection !== 'epsg4326' && formDataObject.mapView.zoom > 4) {
-      errors.mapView.zoom.addError('should be less than or equal to 4')
+  //  Notes: - The validate() function must always return the errors object received as second argument
+  //  The validate() function is called after the JSON schema validation.
+  const customValidate = (formDataObject, errors) => {
```

Comment:
This is currently not working in prod today because the prop at some point was renamed by the package we are using. This fixes it and give the new values what the maximum is supposed to be for open layers

Reply:
We should have a test for this method